pkgs/by-name/in/intentrace/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
```

Comment:
I think the finalAttrs pattern is recommended over `rec`

Reply:
I went with '_let ... in_' pattern to replace rec. Let me know if that meets convention.